servers/src/main/java/tachyon/master/journal/JournalWriter.java

```diff
@@ -143,8 +143,6 @@ public void close() throws IOException {
     if (mEntryOutputStream != null) {
       mEntryOutputStream.close();
     }
-    // Close the ufs.
```

Comment:
Please leave a comment here that the ufs is intentionally not closed, and why it is not closed.